shared/android/app/build.gradle

```diff
@@ -221,6 +227,21 @@ android {
             pickFirst '**/armeabi-v7a/libjsc.so'
         }
     }
+
+    // Explode the Keybase aar for linking against our JSI binding
+    task unzipLibGoJNI(type: Copy) {
```

Comment:
Without this, we can't link against our go library. We want to do that so we can call Go directly from our binding.cpp